src/mapping/LinearCellInterpolationMapping.cpp

```diff
@@ -90,16 +93,21 @@ void LinearCellInterpolationMapping::computeMapping()
     }
   }
 
-  if (!fallbackStatistics.empty() && !missingConnectivity) {
```

Comment:
Would probably be cleaner with two levels of block: `if (!fallbackStatistics.empty())` then check for `hasConnectivity`

Reply:
I don't like nested of blocks, as they somehow make the control flow more complex. However, I adjusted it